httpclient5-fluent/src/main/java/org/apache/hc/client5/http/fluent/Executor.java

```diff
@@ -64,7 +65,9 @@
                         .useSystemProperties()
                         .setMaxConnPerRoute(100)
                         .setMaxConnTotal(200)
-                        .setValidateAfterInactivity(TimeValue.ofSeconds(10))
+                        .setDefaultConnectionMgmtConfig(ConnectionMgmtConfig.custom()
```

Comment:
I would slightly prefer expanding "Mgmt" to "Management" in method and class names for readability, but I don't have a strong opinion if you feel otherwise.

Reply:
@carterkozak I wonder why we need `Mgmt` at all. Do we have `RequestMmgtConfig`. `ConnectionMgmt..` makes sense when it has influence on the connection tool manager. Then it makes sense, but a full word is better to me too.